frontend/godrej-brand-explorer/style.css

```
@@ -0,0 +1,1233 @@
+[data-theme="dark"] {
+    --godrej-light-bg: #0B0F17;
+    --godrej-light-card: #161F2E;
+    --godrej-border-light: #2A364F;
+    --godrej-text-main: #F3F4F6;
+    --godrej-text-muted: #9CA3AF;
+    --godrej-shadow: 0 10px 30px -5px rgba(0, 0, 0, 0.5);
+    --godrej-hover-shadow: 0 20px 35px -5px rgba(0, 85, 165, 0.4);
+}
+
+body {
+    font-family: 'Outfit', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)